README.md

```diff
@@ -13,6 +13,32 @@ This is the R package for simulating spread of pests and pathogens. The package
 
 PoPs is a stochastic spread model of pests and pathogens in forest and agricultural landscapes to learn more visit [popsmodel.org](https://popsmodel.org/). The R package provides an easy way for researchers to calibrate, validate, and test what if scenarios of treatment interventions. The model is also available in GRASS GIS you can install and use [r.pops.spread](https://github.com/ncsu-landscape-dynamics/r.pops.spread) to run the model in GRASS GIS.
 
+## Features
+The PoPS Core C++ library and its interfaces: rpops R package and r.pops.spread GRASS GIS module. The release of rpops includes:
+
+* Susceptible-infected (`SI`) and susceptible-exposed-infected (`SEI`) host phases (`model_type`, `latency_period`),
```

Comment:
Esp. mentioning `latency_period` is definitevely documentation rather than a basic feature list. With that said, parameters and their values make it more concrete which is partially why I included them in r.pops.spread release notes. (The other part is that in notes for subsequent releases, it makes more sense to be specific and say parameter xy renamed/has new option ab etc.) Additionally, in light of limited documentation, I think it is correct to include these here now.